setup.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 
-from distutils.core import setup
+from setuptools import setup
```

Comment:
You need to declare this dependency in the manifest. Otherwise it is not available on the build farm.